src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -185,8 +189,11 @@ public Player generatePlayer() {
         return new Player(generatedId, username, token, team);
     }
 
-    // TODO: Only for testing, feel free to reimplement with corresponding story.
+
     public void addPlayer(Player player) {
+        if (playerMap.size() >= lobbyCapacity) {
+            throw new ResponseStatusException(HttpStatus.CONFLICT, String.format("This lobby is already complete!"));
```

Comment:
I think we were not supposed to use ResponseStatusException in Lobby.
We are supposed to add a separate exception (e.g. FullLobbyException), then capture it in the service and transform it into ResponseSatusException. You can check [this](https://github.com/sopra-fs22-group-16/sopra-fs22-group-16-server/blob/c887cceee896f54494bf36d9ed9497cbf4571453/src/main/java/ch/uzh/ifi/hase/soprafs22/service/LobbyService.java#L160) for an example